### ISSUE TITLE
Scan the memory and error:thread 3: Access violation

### ISSUE THREAD

**Issue by qiuyang1008** [cheat-engine/cheat-engine#618]
When playing the Dragon Quest XI PC Version, and I try to scan the memory of DragonQuest XI game, I instantly get the error information "Scan error:thread 3:Access violation " or "Scan error:thread 2:Access violation " . 

I've just tested that I can successfully scan the memory with cheatengine-i386.exe but it was slow indeed... 

I've been using CE for several years and I've tried to scan the memory of other processes, and it turned out fine. 

My computer: Intel Core i9 7900K and Nvidia Titan XP and 16G*2 memery.

**Comment by cheat-engine** [comment]
and no other information? Like a number behind that ?

**Comment by qiuyang1008** [comment]
There's 2 kinds of number behind that:
1. Scan error:thread 2:Access violation(100)

**Comment by qiuyang1008** [comment]
> and no other information? Like a number behind that ?

OH, Drak, I found it!!!
If the mem_mapped option is unchecked, it will be OK.
If the mem_mapped option is checked, it turn out to be "Scan error:thread 2:Access violation(102) ".
Besides, if the mem_mapped option is checked and CE kernel rountines options are checked, it will be OK but scan slowly and to be unstable(easily crash...)
Hope these may help you. : )

**Comment by cheat-engine** [comment]
what was the valuetype?  A string by any chance ? Longer than 16 characters ?

**Comment by qiuyang1008** [comment]
> what was the valuetype? A string by any chance ? Longer than 16 characters ?

No, 4 bytes. And I test again and any valuetype (such as 4 bytes, float) can reproduce the bug.
If the Scan type mem_mapped option is checked, then it will be an issue. "Scan error:thread 2:Access violation(102) ". Then I uncheck the Scan type mem_mapped option, it will be OK. Strange!
Tested with game "Monster Hunter World" and "Dragon Quest" and other 3 games.
Maybe because the language of my PC system is not English？
